backend/src/main/java/com/google/minicrm/data/Settings.java

```diff
@@ -0,0 +1,127 @@
+package com.google.minicrm.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+public final class Settings implements DatastoreObject {
+    public static String KIND_NAME = "Settings";
+
+    // used for maintaining uniqueness
+    private static long CURRENT_ID = 1;
```

Comment:
Can't do this since we aren't guaranteed the same servlet will be running all the time. For example, if someone makes a setting with id = 1 and then 5 days later another settings object is created, the servlet will not be active all the time and will reset id = 1. Needs to be based off of persistent memory in datastore.